src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -27,17 +22,22 @@ import { BeneficialOwnerOther, BeneficialOwnerOtherKey } from "../../model/benef
 import { ManagingOfficerCorporate, ManagingOfficerCorporateKey } from "../../model/managing.officer.corporate.model";
 import { ManagingOfficerIndividual, ManagingOfficerKey } from "../../model/managing.officer.model";
 
-import { startPaymentsSession } from "../../service/payment.service";
-import { getTransaction } from "../../service/transaction.service";
-import { mapTrustApiReturnModelToWebModel } from "../../utils/trusts";
+import {
+  ID,
+  HasSoldLandKey,
+  NonLegalFirmNoc,
+  IsSecureRegisterKey,
+  OverseasEntityKey,
+  Transactionkey
+} from "../../model/data.types.model";
 
 export const getResumePage = async (req: Request, res: Response, next: NextFunction, resumePage: string) => {
   try {
     logger.debugRequest(req, `GET a saved OE submission`);
 
     const { transactionId, overseaEntityId } = req.params;
     const infoMsg = `Transaction ID: ${transactionId}, OverseasEntity ID: ${overseaEntityId}`;
-    const isRegistration: boolean = req.path.startsWith(config.LANDING_URL);
+    const isRegistration: boolean = isRegistrationJourney(req);
```

Comment:
Again, changes look fine, but given that the resume controller is also now being updated here for the 'check your answers' story I would add a note to testers that the resume journey will require a regression test (registration, update and remove journeys).

Reply:
Every story that is tested is also regression tested with the flag on/off till a submission to chips is successful. This is done with a new submission, a resumed submission, and the update/remove journeys. So this bit will be tested. Thanks.

Reply:
Excellent. Thanks.